bindings/python/crocoddyl/multibody/constraints/frame-velocity-equality.cpp

```diff
@@ -0,0 +1,79 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020, University of Edinburgh, IRI: CSIC-UPC
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "python/crocoddyl/multibody/multibody.hpp"
+#include "crocoddyl/multibody/constraints/frame-velocity-equality.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+void exposeConstraintFrameVelocityEquality() {
+  bp::class_<ConstraintModelFrameVelocityEquality, bp::bases<ConstraintModelAbstract> >(
+      "ConstraintModelFrameVelocityEquality",
+      "This equality constraint function imposes a reference velocity of a given frame, i.e. p - pref = 0, with p "
```

Comment:
It better to substitute p by v since we are talking about velocities

Reply:
done